server/views/reports/show.njk

```diff
@@ -2,11 +2,17 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
 {% from "moj/components/date-picker/macro.njk" import mojDatePicker %}
 {% from "./_reportDataBlock.njk" import reportDataBlock %}
 
 {% extends "../partials/layout.njk" %}
 
+{% block head %}
+  {{super()}}
```

Comment:
Just pulled this down and the linter warned that this should be wrapped in whitespace .e.g 
`{{ super() }}`